workers/check/data_item.py

```diff
@@ -44,7 +46,11 @@ def callback(client_state, channel, method, properties, input_message):
             )
 
             # perform actual action with items
-            processor.do_work(cursor.fetchall())
+            processor.do_work(
+                cursor.fetchall(),
+                do_field_level_checks=is_step_required(settings.FIELD_COVERAGE_STEP),
+                do_resource_level_checks=is_step_required(settings.FIELD_QUALITY_STEP),
```

Comment:
This one is actually `compiled_release`.

Reply:
How do you know that? Which ones are `field_coverage` and `field_quality` then? I'm confused :( (hopefully this won't be confusing anymore after defining the steps' names with the `Steps` class)

Reply:
resource = compiled release
field = field (which is split into coverage and quality)

Reply:
https://pelican-backend.readthedocs.io/en/latest/reference/design.html#levels

Reply:
:+1 aha, and how do you know what file/worker is doing what?